pkg/chartutil/save.go

```diff
@@ -103,14 +102,14 @@ func Save(c *chart.Chart, outDir string) (string, error) {
 	}
 
 	if c.Metadata == nil {
-		return "", errors.New("no Chart.yaml data")
+		return "", ErrChartMetadataMissing
 	}
 
 	cfile := c.Metadata
 	if cfile.Name == "" {
-		return "", errors.New("no chart name specified (Chart.yaml)")
+		return "", ErrChartNameEmpty
```

Comment:
Note that this error message has been changed to be more consistent with other uses of this error. Please let me know if I should revert this.